web-client/src/app.jsx

```diff
@@ -314,7 +314,10 @@ const app = {
     initializeSocketProvider(cerebralApp, applicationContext);
     router.initialize(cerebralApp, route);
 
-    ReactDOM.render(
+    const container = window.document.querySelector('#app');
+    const root = createRoot(container);
+
+    root.render(
```

Comment:
What's this do?

Reply:
this is a change related to a react 18 upgrade.  If you don't change the way we render, internally react will fallback on react 17 logic cc: @rosiehsmith knows more about this

Reply:
yup! we were getting this error in the console locally 
<img width="583" alt="Screen Shot 2022-04-20 at 8 15 22 PM" src="https://user-images.githubusercontent.com/16403861/165319690-da3f9553-62b8-4dac-967c-7dd308c56558.png">
[reference](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis)

Reply:
Oh!  Thanks for pointing that out to me.